engines/advancedDetector.cpp

```diff
@@ -373,7 +373,7 @@ void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFileP
 	g_system->logMessage(LogMessageType::kInfo, reportLog.c_str());
 
 	// Check if the GUI is running, show the UnknownGameDialog and print the translated unknown game information
-	if (GUI::GuiManager::hasInstance() && g_gui.isActive()) {
+       if (GUI::GuiManager::hasInstance() && g_gui.isActive() && useUnknownGameDialog == true) {
 		UnknownGameDialog dialog(report, reportTranslated, bugtrackerAffectedEngine);
```

Comment:
Here as well the indentation seems to have been changed to spaces.

Also the test on `GUI::GuiManager::hasInstance() && g_gui.isActive()` is no longer necessary since I don't see a case where they would fail while `useUnknownGameDialog` is true. But I don't see any issue with keeping those to be extra-safe.

Note that the `== true` is not necessary here either, but since our code formatting conventions don't say anything about this, I guess this is fine.